components/formats-bsd/src/loci/formats/in/OMEXMLReader.java

```diff
@@ -127,6 +127,7 @@ public boolean isThisType(String name, boolean open) {
   public byte[] openBytes(int no, byte[] buf, int x, int y, int w, int h)
     throws FormatException, IOException
   {
+    if (binDataOffsets.size() == 0) return null;
```

Comment:
Could this not return ```buf``` instead of ```null```?  That's what we typically do for missing pixel data, and would eliminate the need for special handling of the ```null``` return value.